ip6defrag/defrag.go

```diff
@@ -0,0 +1,227 @@
+// Package ip6defrag implements a IPv6 defragmenter
+package main
+
+import (
+	"container/list"
+	"fmt"
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
+	"log"
+	"sync"
+	"time"
+)
+
+//need to check
```

Comment:
Spurious comment?  This comment will actually show up in godoc, since it's at the start of a const block, so please fix.  Also, please actually do check :)